### PR TITLE
Add default timeout

### PR DESCRIPTION
**Pull request by fpagnoux**
As suggested by @alexsegura

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+### 24.9.9 [#786](https://github.com/openfisca/openfisca-core/pull/786)
+
+- Set a 120s default timeout in `openfisca serve`
+  - Otherwise, they regularly restart with an anxiety-provoking "[CRITICAL] WORKER TIMEOUT (pid:2137)" message in log
```
Comment (Morendil) [inline CHANGELOG.md]: What will happen instead now?

**Comment by alexsegura** [comment]
Just as an additional note: previously, on Mes Aides, the timeout was set to `60`. 
But now, it is not sufficient anymore. 
It could be great to try to understand what causes the increase in boot time. 
Maybe a bunch of YAML files added recently?

File: CHANGELOG.md (hunk 1 above)
Comment (fpagnoux) [reply CHANGELOG.md]: They should now stay awake and no reboot for no reason

**Comment by fpagnoux** [comment]
> It could be great to try to understand what causes the increase in boot time.
Maybe a bunch of YAML files added recently?

Is the timeout issue really related to the booting time 🤔 ? I had it on the country template, which is really fast to start. 

But if you noticed a slower API boot, I think https://github.com/openfisca/openfisca-core/pull/744 actually had some impact, as the API runs a first calculation when it starts.

**Comment by guillett** [comment]
capture d ecran de 2018-12-03 21-34-27

Here is OpenFisca-Frnace ong booting time explanation.

**Comment by fpagnoux** [comment]
If booting time is important, there are a lot of things that could be loader only on demand ;)

**Comment by guillett** [comment]
To me, it is really hard to _sell_ OpenFisca if the booting time is 20s.

**Pull request merged**